src/Powercord/plugins/pc-commands/injectAutocomplete.js

```diff
@@ -31,7 +31,12 @@ module.exports = async function injectAutocomplete () {
   }
 
   function getMatchingCommand (c) {
-    return [ c.command.toLowerCase(), ...(c.aliases?.map((alias) => alias.toLowerCase()) || []) ];
+    try {
+      return [ c.command.toLowerCase(), ...(c.aliases?.map((alias) => alias.toLowerCase()) || []) ];
+    } catch (e) {
+      console.error('%c[Powercord:Plugin:pc-commands]', 'color: #7289da', e);
```

Comment:
Using [this.error(...)](https://github.com/powercord-org/powercord/blob/v2/src/fake_node_modules/powercord/entities/Plugin.js#L106) would likely be preferred.

Reply:
Just made a new commit that uses the suggestion, I do have a have a question on how well, custom error messages go